specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -283,40 +347,127 @@ not required to upload.
 
   @doc("Optional load test config")
   optionalLoadTestConfig?: OptionalLoadTestConfig;
```

Comment:
OptionalLoadTestConfig -> Naming doesnt look appropriate as the model is LoadTestConfiguration and field name is OptionalLoadTestConfig.